service/bfg/bfg.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2024 Hemi Labs, Inc.
```

Comment:
For files that have been modified, could this be changed to `2024-2025`.
I think the copyright year changes in unmodified files could be reverted as well.